plugins/org.eclipse.osee.ats.ide.integration.tests/src/org/eclipse/osee/ats/ide/integration/tests/ats/query/AtsActionEndpointImplTest.java

```diff
@@ -254,6 +254,7 @@ public void testAtsActionsDetailsRestCall() {
       Assert.assertEquals(action.get("AtsId").asText(), action.get("ats.Id").asText());
       Assert.assertFalse(Strings.isNumeric(action.get("ats.Created Date").asText()));
       Assert.assertNotNull(action.get("derivedfrom"));
```

Comment:
Nit: Should this be camel case? "derivedFrom"

Reply:
same